src/error-handler.js

```diff
@@ -67,9 +67,46 @@ function primitiveErrorModal(enduser_message, github_issue_url, mailto_url) {
     document.querySelector('#error-modal .close-button').onclick = dismiss;
 }
 
+const flash_message_id = 'flash-message-error-online';
+const button_id = 'button-flash-message-error-online';
 try {
     const handler = (event) => {
         try {
+            if (event.error.message === 'Network Error') {
```

Comment:
Most things I have tried (submitting comments, donations, loading templates in the generator) produce another error and are thus not caught by the current code.

In Chrome, the message seems to be "Uncaught TypeError: Failed to fetch" and in Firefox, it seems to be "TypeError: NetworkError when attempting to fetch resource.".